### PR TITLE
`module_generator` bugfix to choose correct atmosphere extension.

### PR DESCRIPTION
**Pull request by DevarshiChoudhury**

### CODE REVIEW

File: xpsi/module_generator.py [hunk 1]
```diff
@@ -1930,8 +1930,10 @@ def get_member_settings(model):
 get_member_settings(args.hot_region_model[0])
 get_bounds_and_values(args.prefix[0], args.hot_region_model[0])
 
-module += (
-'''
+if 'NSX' in args.hot_atmosphere_model or 'nsx' in args.hot_atmosphere_model:
+    print(args.hot_atmosphere_model)
```
Comment (thjsal) [inline xpsi/module_generator.py:1934]: Is this printing accidentally left from debugging to here, or purposely?

**Comment by thjsal** [comment]
The fixes look good to me. I also looked at the elsewhere option in module generator (as you mentioned in the original issue), and looks like that no atmosphere selection has been implemented to that yet. So it uses always the default blackbody atmosphere.

**Comment by DevarshiChoudhury** [comment]
@thjsal print statement removed and option for elsewhere added. Tested it and seems to work fine

**Review by thjsal — APPROVED** [review]
Looks good to me!

**Pull request merged**